lib/testers/rule-tester.js

```diff
@@ -398,6 +398,7 @@ class RuleTester {
         function assertASTDidntChange(beforeAST, afterAST) {
 
             // Feature detect the Node.js implementation and use that if available.
+            // eslint-disable-next-line node/no-unsupported-features/node-builtins
```

Comment:
I'm a bit confused about this line. Is the name of the rule `"no-unsupported-features/node-builtins"` (and therefore the plugin-qualified name is `"node/no-unsupported-features/node-builtins"`? I don't think I've ever seen a rule defined with a forward slash in its name, just wanted to make sure I understood correctly.

Reply:
Yes. We can use `/` in the rule name in plugins. https://github.com/eslint/eslint/issues/3458#issuecomment-257161846 mentioned this fact.